packer/configure-windows-vhd.ps1

```diff
@@ -83,8 +83,8 @@ function Get-FilesToCacheOnVHD
             "https://acs-mirror.azureedge.net/wink8s/v1.17.0-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://acs-mirror.azureedge.net/cni/azure-vnet-cni-windows-amd64-v1.0.28.zip",

```

Comment:
Let's keep 1.0.28 in the VHD to allow folks to use the VHD who don't want to necessarily update to 1.0.30 or 1.0.29

Reply:
+1

Reply:
added it back